pkg/repo/guild_user_activity_log/pg.go

```diff
@@ -16,3 +16,7 @@ func NewPG(db *gorm.DB) Store {
 func (pg *pg) CreateOne(record model.GuildUserActivityLog) error {
 	return pg.db.Create(&record).Error
 }
+
+func (pg *pg) CreateOneNoGuild(record model.GuildUserActivityLog) error {
```

Comment:
Why need this ?
CreateOne can also create a record with null guild id

Reply:
string values cannot be null, it will default to `""` which is an error for foreign keys

Reply:
agree